main.cpp

```diff
@@ -13,75 +13,506 @@ What you create in this project will be used as the basis of Project 5 in the co
+    2) inner material // string
+    3) how many riders it can seat // int
+    4) comfort rating // double
+    5) safety rating // double
+3 things it can do:
+    1) seat 2 riders at the same time
+    2) repel rain with its waterproof material
+    3) provide comfort for the rider
+ */
+
+struct Seat
+{
+    // outer material
+    std::string outerMaterial = "leather";
+    // inner material
+    std::string innerMaterial = "polyster";
+    // how many riders it can seat
+    int riders = 1;
+    // comfort rating
+    double comfortRating = 6.5;
+    // 5) safety rating 
+    double safetyRating = 9.5;
+
+    // seat riders
+    void seatRiders(int riders);
+    // repel rain with its waterproof material
+    void waterproof(bool applyWaterProofCoatToLeatherExterior = true);
+    // provide comfort for the rider
+    void comfort(int riders, double combinedWeightKG);
+};
+
+/*
+Thing 6) Engine
+5 properties:
+    1) pistons // int
+    2) valves // int
+    3) cyclinder head manufacturer // string
+    4) quanity of oil pans // int
+    5) engine block type // string
+3 things it can do:
+    1) produce energy for the motorcycle to move
+    2) lubricate itself with oil to keep the parts moving fluidly
+    3) burn fuel
+ */
+
+struct Engine
+{
+    // pistons
+    int pistons = 2;
+    // valves
+    int valves = 16;
+    // cyclinder head  manufacturer
+    std::string cyclinderHeadManufacturer = "Suzuki";
+    // quanity of oil pans
+    int oilPans = 2;
+    // engine block type
+    std::string engineBlockType = "Combustion";
+
+    // produce energy for the motorcycle to move
+    void combust(int pistons, int valves, std::string engineBlockType);
+    // lubricate itself with oil to keep the parts moving fluidly
+    void lubricate(int oilPans, bool lubricateEngine);
+    // burn fuel
+    void burnFuel(bool burnFuel = true, bool regulateFuelIntake = true);
+};
+
+/*
+Thing 7) Exhaust
+5 properties:
+    1) length in inches // double
+    2) turbocharger power // double
+    3) catalytic converters // int
+    4) muffler type // string
+    5) number of pipes // int
+3 things it can do:
+    1) increase the power of the engine using a turbocharger
+    2) reduce fumes using catalytic converters
+    3) reduce or increase the level of noise when reving the engine
+ */
+
+struct Exhaust
+{
+    // length in inches
+    double lengthinInches = 15.4;
+    // turbocharger power
+    double turboChargerPower = 4.5;
+    // catalytic converters
+    int catalyticConverters = 1;
+    // muffler type
+    std::string mufflerType = "High Supression";
+    // number of pipes
+    int pipes = 2;
+
+    // increase the power of the engine using a turbocharger
+    void turboChargeOutput(double turboChargerPower);
+    // reduce fumes using catalytic converters
+    void catalyticConversion(bool performCatalyticConversion = true);
+    // reduce or increase the level of noise when reving the engine
+    void levelOfNoise(std::string mufflerType);
```

Comment:
none of these names contain the 'action' or verb.
change your function names to match the plain-english description
- [ ] revise